src/app/pages/account/account-detail/account-detail-table/account-detail-table.component.html

```diff
@@ -40,7 +40,7 @@
                           [content]="data[template.matColumnDef] | formatDigit: 2">
                         </app-tooltip-customize>
                       </div>
-                      <span class="text--white ml-1"> {{ denom }}</span>
+                      <span class="text--white ml-1" [appLinkDenom]> {{ denom }}</span>
```

Comment:
Remove `text--white` ở đây luôn nhé, không cần phải remove ở Diretive
```suggestion
                      <span class="ml-1" [appLinkDenom]> {{ denom }}</span>
```

Reply:
Chỗ này fix thiếu rồi, những chõ nào không cần input thì không cần thêm `[]`
`[appLinkDenom]` -> `appLinkDenom`